src/main/java/org/isf/utils/db/DbSingleConn.java

```diff
@@ -37,15 +37,11 @@
 
 import com.mysql.jdbc.exceptions.jdbc4.CommunicationsException;
 
-/*
- * @version 0.1 2005-11-06
- * @author bob
- *
- */
-
 /**
- * classe singleton che provvede alla connessione con database i parametri: dati
+ * Classe singleton che provvede alla connessione con database i parametri: dati
```

Comment:
Please change it in English
"Singleton class that provides connection with the database"

Reply:
Now in English